app/models/game.rb

```diff
@@ -49,6 +49,10 @@ def populate_board
     Rook.create(game_id: id, user_id: black_player_id, row: 7, col: 7, is_black: true)
   end
 
+  def blacks_turn?
+    !moves.last.piece.is_black
```

Comment:
Who moves first? A specific side always moves first or we randomly determine it?

In the initial game situation where moves.empty? returns true - what will blacks_turn? return - a true or a false.